README.md

```diff
@@ -99,8 +99,34 @@ See the [Jupyter notebooks](https://github.com/qctrl/notebooks/tree/master/qctrl
 
 ## Contributing
 
-See
-[Contributing](https://github.com/qctrl/.github/blob/master/CONTRIBUTING.md).
+For general guidelines, see [Contributing](https://github.com/qctrl/.github/blob/master/CONTRIBUTING.md).
+
+### Building documentation
+
+Documentation generation relies on [Spinx](http://www.sphinx-doc.org). Automated builds are done by [Read The Docs](https://readthedocs.com).
+
+To build locally:
+
+1. Ensure you have used one of the install options above.
+1. Ensure you have the environment variable APITOKEN set to a valid token.
```

Comment:
Update numbering

Reply:
Actually it renders fine. This was a standards discussion a while ago.

Reply:
Mind blown, thanks!